env.sample

```diff
@@ -7,4 +7,7 @@ MIT_TR_URL=http://mit-tr.staging.terarium.ai
 LOG_LEVEL=INFO
 MOCK_TA1=True
 MOCK_TDSt=True
-OPENAI_API_KEY=foo
\ No newline at end of file
+OPENAI_API_KEY=foo
```

Comment:
I removed the need to provide the open.ai key to MIT_TR, it is now embedded.

Reply:
ah ok we may need to update some endpoints to remove it as a parameter if they no longer want it

Reply:
They still want it for sure. I just rewrote their service in our container to add it as an ENV variables on startup. We should do a clean up on TDS and KM [once we move to our image](https://github.com/DARPA-ASKEM/orchestration/pull/279)